src/components/Section/Section.js

```diff
@@ -20,9 +20,10 @@ export const Section = memo(({
   isOpen,
   className,
   children,
+  isVisible = true,
   ...rest
 }) => {
-  const [isChecked, setChecked] = useState(optional ? isOpen : true);
+  const [isChecked, setChecked] = useState(optional ? isOpen : isVisible);
```

Comment:
Doesn't `isOpen` prop have the same meaning as `isVisible`? Can we change the line [here](https://github.com/folio-org/ui-data-import/pull/513/files#diff-6709362c6d184124da81079fe4cef458R492) to `isOpen: hasData` e.g. and then `isVisible` prop will not be needed?